### DISCUSSION TITLE
PullAprove

### DISCUSSION THREAD

**Discussion by maggu2810** [openhab/openhab-core#3496] [Core Maintainers Discussions]
openhab2-addons already uses PullApprove, so I don't it will harm to use it for openHAB Core, too.
It can be used to make the "Signed-off-by" requirements more visible.

I requested to add it:

> A request to install PullApprove has been submitted on the @openhab account. 

Who needs to have a look at?

**Comment by kaikreuzer** [comment]
I received the request and approved it.
